libraries/ms-common/src/lights_signal_fsm.c

```diff
@@ -80,17 +104,43 @@ static void prv_state_hazard_signal_output(Fsm *fsm, const Event *e, void *conte
   blink_event_generator_start(&storage->blink_event_generator, storage->signal_hazard_output_event);
 }
 
+static void prv_blink_event_raised_callback(BlinkerState new_state, void *context) {
+  SignalFsmStorage *storage = context;
+
+  // we only send sync events on rising edges
+  if (new_state == BLINKER_STATE_ON) {
+    storage->blink_counter++;
```

Comment:
isn't the `blink_counter` only used by the sync event sender? should the increment be inside the following if statement?